compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -1196,6 +1205,7 @@ class Namer { typer: Typer =>
       cls.setNoInitsFlags(parentsKind(parents), untpd.bodyKind(rest))
       if (cls.isNoInitsClass) cls.primaryConstructor.setFlag(StableRealizable)
       processExports(using localCtx)
+      index(restAfterParents)(using localCtx)
```

Comment:
Not nice to have to do another index here.

Reply:
this is no longer part of the implementation